_pages/home/release-information.md

```diff
@@ -265,8 +264,17 @@ For both patch and minor releases, upgrading to the new libraries should not cau
 
 We don't plan to introduce a new major release that frequently, unless an issue is found that makes it necessary to move to a major release . The upcoming Angular "Ivy" (after Angular 8) is one factor in our eventual decision to do so.
   
-  
-  
+### Upgrading Spartacus Libraries to a New Minor Version
+
+To upgrade your Spartacus libraries to a new minor version, set your @spartacus libraries in `package.json` to “~1.#.0" (replace `#` with the release number you wish to upgrade to). Then delete your `node_modules` folder and run `yarn install`.
```

Comment:
Nothing wrong with this paragraph, but could we perhaps cut it since the way you rephrased it below is nice and clear?

Reply:
Yes, I was supposed to delete this section, woops! 😄 
Thank you for catching that!